src/Components/Server/src/Circuits/RemoteJSRuntime.cs

```diff
@@ -70,12 +70,12 @@ private void EndInvokeDotNetCore(string callId, bool success, object resultOrErr
 
         protected override void BeginInvokeJS(long asyncHandle, string identifier, string argsJson)
         {
-            if (!_clientProxy.Connected)
+            if (_clientProxy is null)
```

Comment:
`_clientProxy` isn't initiailized during pre-rendering and this change reflects that. Knowing how unreliable the the `Connected` flag is, it made sense to avoid checking it here. It makes the experience with JSInterop a bit more consistent - users should now consistently see timeouts from JSInterop calls when the client send failed, rather than a timeout sometime and a sync exception at other times.